Public/Src/Engine/Scheduler/PipExecutor.cs

```diff
@@ -2319,22 +2347,40 @@ async Task<RunnableFromCacheResult> innerCheckRunnableFromCacheAsync(List<BoxRef
                             // or a mismatch of strong fingerprints (at least one ref checked).
                             if (numCacheEntriesVisited == 0)
                             {
-                                pipCacheMiss.Value.CacheMissType = PipCacheMissType.MissForDescriptorsDueToWeakFingerprints;
-                                Logger.Log.TwoPhaseCacheDescriptorMissDueToWeakFingerprint(
-                                    operationContext,
-                                    description,
-                                    weakFingerprint.ToString());
+                                if (isWeakFingerprintAugmented)
+                                {
+                                    pipCacheMiss.Value.CacheMissType = PipCacheMissType.CacheMissesForDescriptorsDueToAugmentedWeakFingerprints;
+                                    Logger.Log.TwoPhaseCacheDescriptorMissDueToAugmentedWeakFingerprint(

```

Comment:
>TwoPhaseCacheDescriptorMissDueToAugmentedWeakFingerprint [](start = 47, length = 56)

Suggest removing this, TwoPhaseCacheDescriptorMissDueToWeakFingerprint(), and TwoPhaseCacheDescriptorMissDueToStrongFingerprints(). See other comment above

Reply:
Discussed offline


---
In reply to: [347055775](https://github.com/microsoft/BuildXL/pull/1205#discussion_r347055775) [](ancestors = 347055775)